sherpa/bin/pruned_stateless_emformer_rnnt2/streaming_client.py

```diff
@@ -57,6 +58,7 @@ def get_args():
     parser.add_argument(
         "sound_file",
         type=str,
+        required=True,
```

Comment:
```suggestion
```

Reply:
this should be fine, I think -- I think it collides with nargs='+' parameter? I'm still reformating because of pre-commit, so I will investigate